changelog/unreleased/issue-7729

```diff
@@ -0,0 +1,9 @@
+Change: Update to Qt-5.12.7
```

Comment:
Change: Bundle Qt 5.12.7

We updated the bundled version of Qt tot 5.12.7.
This resolves the following issues:
* plugins were picked up from the current directory.

https://github.com/owncloud/client/issues/7729
https://github.com/owncloud/client/issues/7567